tests/test_writer.py

```diff
@@ -236,6 +236,10 @@ def writer_operate(writer: PdfWriter) -> None:
     writer.page_mode = NameObject("/UseOC")
     assert writer._get_page_mode() == "/UseOC"
     writer.insert_blank_page(width=100, height=100)
+    page = writer.insert_blank_page(width=100)
+    assert page.mediabox.height == 100
```

Comment:
Please use a dedicated test method for this and add assertions for the width and height respectively as well to ensure that it is indeed working correctly.